functions/lib/crowd_webhook.js

```diff
@@ -0,0 +1,87 @@
+module.exports = CrowdWebhook
+
+function hasProperty (object, property) {
+  return Object.prototype.hasOwnProperty.call(object, property)
+}
+
+function CrowdWebhook (groupMappings, github) {
+  if (!groupMappings) {
+    // Use default mapping if one is not provided
+    groupMappings = require('./crowd_webhook_config')
```

Comment:
I think we should not use a default config file. If groupMappings is missing that should be an error.